src/components/new-request/search.vue

```diff
@@ -134,7 +61,6 @@
                 :label="getIsConversion ? 'Select type of business to alter into' : 'Select type of business in B.C.'"
                 :error-messages="getErrors.includes('entity_type_cd') ? 'Please select a business type' : ''"
                 :items="entityConversionTypeOptions"
-                :class="!getLocation ? 'disabled-custom' : ''"
```

Comment:
Don't need this since menu is not rendered until needed (progressive disclosure).